zh/configure-a-tidb-cluster.md

```diff
@@ -671,6 +686,46 @@ affinity:
        - ${namespace}
 ```
 
+#### 通过 topologySpreadConstraints 实现均匀分布
```

Comment:
https://docs.pingcap.com/zh/tidb-in-kubernetes/dev/deploy-tidb-dm also needs to check.